lib/iris/analysis/_interpolate_private.py

```diff
@@ -206,6 +206,11 @@ def _nearest_neighbour_indices_ndcoords(cube, sample_points, cache=None):
     Because this function can be slow for multidimensional coordinates,
     a 'cache' dictionary can be provided by the calling code.
 
+    .. Note::
+
+        If the points are longitudes/latitudes, these are handled correctly as
+        points on the sphere, but the values must be in 'degrees'.
```

Comment:
> is this protected somewhere? if i supply radians, will a meaningful exception be raised?

No, it is not checked and will just give wrong answers !

I decided to document this problem rather than fix it, as it is an existing low-level routine.
As this is a private routine I thought that was ok.
Arguably, we could maybe fix the existing analysis.trajectory.interpolate, as it calls this and so exposes the same problem.
? what do you think @marqh ?

I **have** fixed it in the UnstructuredNearest classes, which now convert everything to degrees (and complain if it fails).